### PR TITLE
Fixed ranged enemy

### PR DESCRIPTION
**Pull request by LeoWikstrom**
Made ranged enemy projectile disappear after one hit. Also buffed ranged enemy, since he was terribly weak, and slightly increased velocity of player projectile

### CODE REVIEW

File: StortSpelProjekt/Game/src/main.cpp [hunk 1]
```diff
@@ -414,7 +415,7 @@ Scene* GameScene(SceneManager* sm)
 
     EnemyFactory::GetInstance().DefineEnemy("enemyZombie", &zombie);
     EnemyFactory::GetInstance().DefineEnemy("enemySpider", &spider);
-    EnemyFactory::GetInstance().DefineEnemy("enemyDemon", &rangedDemon);
```
Comment (AntonAsbrink) [inline StortSpelProjekt/Game/src/main.cpp]: Changed define enemy causing no demons to be spawned

**Review by AntonAsbrink — DISMISSED** [review]

File: StortSpelProjekt/Game/src/main.cpp (hunk 1 above)
Comment (LeoWikstrom) [reply StortSpelProjekt/Game/src/main.cpp]: That's a biiig oopsie 😓

File: StortSpelProjekt/Vendor/Resources/ShopRoom.map [hunk 2]
```diff
@@ -273,6 +273,19 @@ ModelLightAttenuation 1.0, 0.22, 0.20
...
+ModelRotation 0.0, 0.0, 0.0
+ModelPosition -83.0, 15.0, 25.0
+ModelDrawFlag 1, 1
+ModelMass 0.0
+ModelFriction 1.0
+ModelRestitution 0.0
+Submit Model
+Submit CollisionCube 1.0, 1.0, 1.0
+AddToEdge 4
+
```
Comment (Schtekt) [inline StortSpelProjekt/Vendor/Resources/ShopRoom.map:288]: Sneaky

**Review by Schtekt — APPROVED** [review]
All good!

**Review by AntonAsbrink — APPROVED** [review]

**Pull request merged**